### PR TITLE
Update api-explorer.md

### PR DESCRIPTION
**Pull request by msbemba**
Added info on how to access API explorer

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8705

File: microsoft-365/security/defender-endpoint/api-explorer.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ With the API Explorer, you can:
 
 ## Access API Explorer
 
-From the left navigation menu, select **Partners & APIs** \> **API Explorer**.
+From the left navigation menu , select **Partners & APIs** \> **[API Explorer](https://security.microsoft.com/interoperability/api-explorer)**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/api-explorer.md]:
```suggestion
From the left navigation menu, select **Partners & APIs** \> **[API Explorer](https://security.microsoft.com/interoperability/api-explorer)**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Gentle reminder

**Pull request merged**